code/modules/reagents/reagent_containers/hypospray.dm

```diff
@@ -162,6 +162,14 @@
 	amount_per_transfer_from_this = 42
 	list_reagents = list("salbutamol" = 10, "teporone" = 15, "epinephrine" = 10, "lavaland_extract" = 2, "weak_omnizine" = 5) //Short burst of healing, followed by minor healing from the saline
 
+/obj/item/reagent_containers/hypospray/autoinjector/emergency_nuclear
+	name = "emergency stabilization medipen"
+	desc = "A fast acting livesaving emergency autoinjector. Effective in combat situations, make by the syndicate for the syndicate.</span>"
```

Comment:
I don't think you meant to have the /span there, correct me if I'm wrong.
```suggestion
	desc = "A fast acting live-saving emergency autoinjector. Effective in combat situations, made by the syndicate for the syndicate."
```

Reply:
span was accidental

Reply:
oh I typoed here too, I meant `life-saving`, oops.

Reply:
fixed